service/src/main/kotlin/io/provenance/explorer/service/ValidatorService.kt

```diff
@@ -354,12 +354,15 @@ class ValidatorService(
     fun getProposerConsensusAddr(blockMeta: Query.GetBlockByHeightResponse) =
         blockMeta.block.header.proposerAddress.translateByteArray(props).consensusAccountAddr
 
-    fun saveProposerRecord(blockMeta: Query.GetBlockByHeightResponse, timestamp: DateTime, blockHeight: Int) =
-        transaction {
-            val consAddr = getProposerConsensusAddr(blockMeta)
-            val proposer = findAddressByConsensus(consAddr)!!.operatorAddress
+    fun saveProposerRecord(blockMeta: Query.GetBlockByHeightResponse, timestamp: DateTime, blockHeight: Int) = let {
+        val consAddr = getProposerConsensusAddr(blockMeta)
+        val proposer = findAddressByConsensus(consAddr)!!.operatorAddress
+        try {
+            BlockProposerRecord.save(blockHeight, null, timestamp, proposer)
```

Comment:
Why if there is an error is BlockProposerRecord.save() called again with the same inputs?  What would cause it to not fail the second time?

Reply:
Looks like this is just pulling a try catch out of the save() function that automatically retried if it failed.  I would be curious to know why that is there, but this is fine for this pr.